src/notelistview.cpp

```diff
@@ -929,14 +929,17 @@ QPixmap NoteListViewPrivate::renderToPixmap(const QModelIndexList &indexes, QRec
         option.rect = paintPairs.at(j).rect.translated(-r->topLeft());
         const QModelIndex &current = paintPairs.at(j).index;
 #endif
+        Q_Q(const QAbstractItemView);
         adjustViewOptionsForIndex(&option, current);
-        delegateForIndex(current)->paint(&painter, option, current);
+        q->itemDelegateForIndex(current)->paint(&painter, option, current);
```

Comment:
Still not sure what this actually does, but this fixes the build. Still needs to be thoroughly tested.